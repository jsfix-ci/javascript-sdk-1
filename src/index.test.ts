import 'jest';
import * as Exports from '.';

describe('package exports', () => {
    it('should export public symbols', () => {
        expect(Exports).toMatchSnapshot();
    });
});
