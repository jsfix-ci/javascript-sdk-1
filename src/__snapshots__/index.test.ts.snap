// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`package exports should export public symbols 1`] = `
Object {
  "CampaignLifecycleStatus": Object {
    "DRAFT": "draft",
    "PENDING": "pending",
    "SCHEDULED": "scheduled",
    "SENT": "sent",
  },
  "ContactType": Object {
    "ORGANISATION": "organisation",
    "PERSON": "person",
  },
  "CoverageProvider": Object {
    "AMMCO": "ammco",
    "AUXIPRESS": "auxipress",
    "BELGA": "belga",
    "FEED": "feed",
    "KANTAR": "kantar",
    "KNEWIN": "knewin",
    "MANUAL": "manual",
    "MEDIAWEB": "mediaweb",
    "MELTWATER": "meltwater",
  },
  "CoverageType": Object {
    "ONLINE": "online",
    "PRINT": "print",
    "RADIO": "radio",
    "SOCIAL": "social",
    "TELEVISION": "television",
  },
  "EmailAddressStatus": Object {
    "BOUNCED": "bounced",
    "OK": "ok",
    "UNSUBSCRIBED": "unsubscribed",
    "UNSUBSCRIBED_FROM_ALL_COMMUNICATIONS": "unsubscribed_from_all_communications",
  },
  "EmailBrandingMode": Object {
    "CUSTOM": "custom",
    "DEFAULT": "default",
  },
  "Gender": Object {
    "FEMALE": "female",
    "MALE": "male",
    "UNSPECIFIED": "unspecified",
  },
  "JobStatus": Object {
    "PENDING": "pending",
    "REJECTED": "rejected",
    "RESOLVED": "resolved",
  },
  "MediumType": Object {
    "ANALYST": "analyst",
    "BLOG": "blog",
    "EXPERT": "expert",
    "INFLUENCER": "influencer",
    "MAGAZINE": "magazine",
    "NEWSLETTER": "newsletter",
    "NEWSPAPER": "newspaper",
    "NEWS_SITE": "news website",
    "PHOTO": "photo",
    "PODCAST": "podcast",
    "PRINT": "print",
    "RADIO": "radio",
    "SOCIAL": "social",
    "TV": "tv",
    "VLOG": "vlog",
    "WEB": "web",
    "WIRE": "wire",
  },
  "NewsroomWebhookEvent": Object {
    "CATEGORY_CREATED": "category.created",
    "CATEGORY_DELETED": "category.deleted",
    "CATEGORY_UPDATED": "category.updated",
    "STORY_CREATED": "story.created",
    "STORY_DELETED": "story.deleted",
    "STORY_UPDATED": "story.updated",
  },
  "NotificationStyle": Object {
    "DANGER": "danger",
    "INFO": "info",
    "SUCCESS": "success",
    "WARNING": "warning",
  },
  "Periodicity": Object {
    "ANNUALLY": "annually",
    "BI_ANUALLY": "bi-anually",
    "BI_MONTHLY": "bi-monthly",
    "BI_WEEKLY": "bi-weekly",
    "DAILY": "daily",
    "MONTHLY": "monthly",
    "QUARTERLY": "quarterly",
    "SEMI_ANNUALLY": "semi-annually",
    "WEEKDAYS": "weekdays",
    "WEEKLY": "weekly",
  },
  "PhoneNumberType": Object {
    "CELLPHONE": "cell",
    "TELEPHONE": "tel",
  },
  "SelectionMode": Object {
    "ALL": "all",
    "IN": "in",
    "NOT_IN": "not_in",
  },
  "SenderAddressType": Object {
    "CUSTOM": "custom",
    "PREZLYMAIL": "prezly_mail",
  },
  "SenderDomainVerificationFlowVersion": Object {
    "V1": "v1",
    "V2": "v2",
    "V3": "v3",
  },
  "SenderDomainVerificationStatus": Object {
    "INVALID": "invalid",
    "NOT_APPLICABLE": "not-applicable",
    "UNKNOWN": "unknown",
    "VALID": "valid",
  },
  "SocialNetwork": Object {
    "DISCORD": "discord",
    "FACEBOOK": "facebook",
    "INSTAGRAM": "instagram",
    "LINKEDIN": "linkedin",
    "PINTEREST": "pinterest",
    "SKYPE": "skype",
    "SNAPCHAT": "snapchat",
    "TWITCH": "twitch",
    "TWITTER": "twitter",
    "YOUTUBE": "youtube",
  },
  "StoryAlignment": Object {
    "CENTER": "center",
    "LEFT": "left",
    "RIGHT": "right",
  },
  "StoryAppearance": Object {
    "FULL": "full",
    "INTRO": "intro",
  },
  "StoryFormatVersion": Object {
    "1": "HTML",
    "3": "SLATEJS",
    "HTML": 1,
    "SLATEJS": 3,
  },
  "StoryLifecycleStatus": Object {
    "DRAFT": "draft",
    "EMBARGO": "embargo",
    "PUBLISHED": "published",
    "SCHEDULED": "scheduled",
    "UNINITIALIZED": "uninitialized",
  },
  "StoryPublicationStatus": Object {
    "DRAFT": "draft",
    "NEW": "new",
    "PUBLISHED": "published",
  },
  "StoryVisibility": Object {
    "EMBARGO": "embargo",
    "PRIVATE": "private",
    "PUBLIC": "public",
  },
  "TextDirection": Object {
    "LTR": "ltr",
    "RTL": "rtl",
  },
  "ThemeFeature": Object {
    "BOILERPLATE": "boilerplate",
    "CATEGORIES": "categories",
    "CUSTOM_COLORS": "custom_colors",
    "MEDIA_GALLERIES": "media_galleries",
    "MULTI_LANGUAGE": "multi_language",
    "OPTIONAL_PRIVACY_PORTAL": "optional_privacy_portal",
    "SEARCH": "search",
  },
  "UnsubscribeReason": Object {
    "NEVER_SIGNED_UP": "never_signed_up",
    "OTHER": "other",
    "SPAM": "spam",
    "UNKNOWN": "unknown",
    "UNWANTED": "unwanted",
  },
  "default": [Function],
}
`;
